agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -669,3 +669,66 @@ func TestMetadataServiceValidator(t *testing.T) {
 
 	assert.Equal(t, 42, exitCode, fmt.Sprintf("Expected exit code of 42; got %d", exitCode))
 }
+
+// TestExecutionRole verifies that task can use the execution credentials to pull from ECR and
+// send logs to cloudwatch with awslogs driver
+func TestExecutionRole(t *testing.T) {
+	if os.Getenv("TEST_EXECUTION_ROLE") != "true" {
+		t.Skip("TEST_EXECUTION_ROLE was not set to true")
+	}
+
+	RequireDockerVersion(t, ">=17.06.2-ce") // awslogs drivers with execution role available from docker 17.06.2
```

Comment:
didn't you mention in another PR that `-ce` would not be expected? do we need it here?

Reply:
Actually we needed it, I tested that the `17.06.2-ce>=17.06.2` will return false, I have modified in line 674 also.